src/vcpkg/commands.add-version.cpp

```diff
@@ -29,11 +29,91 @@ namespace
                                  (msg::new_scheme, msg::old_scheme, msg::package_name, msg::option),
                                  "",
                                  "Use the version scheme \"{new_scheme}\" instead of \"{old_scheme}\" in port "
                                  "\"{package_name}\".\nUse `--{option}` to disable this check.");
+    DECLARE_AND_REGISTER_MESSAGE(AddVersionVersionAlreadyInFile,
+                                 (msg::version, msg::path),
+                                 "",
+                                 "version {version} is already in {path}");
+    DECLARE_AND_REGISTER_MESSAGE(AddVersionAddedVersionToFile,
+                                 (msg::version, msg::path),
+                                 "",
+                                 "added version {version} to {path}");
+    DECLARE_AND_REGISTER_MESSAGE(AddVersionNewFile, (), "", "(new file)");
+    DECLARE_AND_REGISTER_MESSAGE(AddVersionPortFilesShaUnchanged,
+                                 (msg::version, msg::path),
+                                 "",
+                                 "local port files SHA is the same as version {version} in {path}");
+    DECLARE_AND_REGISTER_MESSAGE(AddVersionCommitChangesReminder, (), "", "Did you remember to commit your changes?");
+    DECLARE_AND_REGISTER_MESSAGE(AddVersionNoFilesUpdated, (), "", "No files were updated");
+    DECLARE_AND_REGISTER_MESSAGE(AddVersionNoFilesUpdatedForPort,
+                                 (msg::package_name),
+                                 "",
+                                 "No files were updated for {package_name}");
+    DECLARE_AND_REGISTER_MESSAGE(AddVersionPortFilesShaChanged,
+                                 (msg::package_name),
+                                 "",
+                                 "local changes detected for {package_name} but no changes to version or port version");
+    DECLARE_AND_REGISTER_MESSAGE(AddVersionVersionIs, (msg::version), "", "version: {version}");
+    DECLARE_AND_REGISTER_MESSAGE(AddVersionOldShaIs,
+                                 (msg::value),
+                                 "{value} is a 40-digit hexadecimal SHA",
+                                 "old SHA: {value}");
+    DECLARE_AND_REGISTER_MESSAGE(AddVersionNewShaIs,
+                                 (msg::value),
+                                 "{value} is a 40-digit hexadecimal SHA",
+                                 "new SHA: {value}");
+    DECLARE_AND_REGISTER_MESSAGE(AddVersionUpdateVersionReminder,
+                                 (),
+                                 "",
+                                 "Did you remember to update the version or port version?");
+    DECLARE_AND_REGISTER_MESSAGE(AddVersionOverwriteOptionSuggestion,
+                                 (),
+                                 "",
+                                 "Use --overwrite-version to bypass this check");
```

Comment:
The option name should be passed in, since it should not be localized.

(`"Use --{option} ...."`)